domains/eduardoaugusto.json

```
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "d3veduardo",
+    "email": "eduardoaugusto-is-a-dev@outlook.com"
+  },
+  "records": {
+    "CNAME": "cname.squareweb.app",
+    "TXT": "HRcY7tueGTU8dwHbS5xCj7hnoZYLOKASeQVOGq4H_M8"
+  },
+  "proxied": false
```

you cant combine CNAME with other records
```suggestion
  "proxied": true
```